lib/upgrade_lts_contract.py

```diff
@@ -13,6 +14,8 @@
 
 if __name__ == "__main__":
     setup_logging(logging.DEBUG)
+    gettext.bindtextdomain("ubuntu-pro", "/usr/share/locale")
```

Comment:
Why do we need that call ? Aren't we using the default path location to store the translations ?

Reply:
The path is the default, but all examples of python apps in ubuntu that I've looked at pass this value to this call anyway, so I'm following their example.

(Note this has changed to a slightly different call at the top of `messages/__init__.py` but still hardcodes this path)